pkg/zfs/zfs_util.go

```diff
@@ -46,14 +46,14 @@ const (
 
 // constants to define volume type
 const (
-	VOLTYPE_DATASET = "DATASET"
-	VOLTYPE_ZVOL    = "ZVOL"
+	VoltypeDataset = "DATASET"
```

Comment:
VolTypeDataset and VolTypeZvol?

Reply:
ya .. i was confused if this is something I did ... or not . sorry